Examples/Nodify.Shapes/Canvas/Gestures/UnboundGestureMappings.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Nodify.Shapes.Canvas
+{
+    public class UnboundGestureMappings : EditorGestures
+    {
+        public static readonly UnboundGestureMappings Instance = new UnboundGestureMappings();
+
+        public UnboundGestureMappings()
+        {
+            Editor.Selection.Apply(SelectionGestures.None);
```

Comment:
This is a nice approach, definitly better then binding IsReadOnly/IsLocked all all over the place, i thought about it and thats the best yeah, would be nice if the same is also adjustable in a instanced way for the misc actions around editing e.g.:

- currently undo/redo is possible while locked.
- the nodes/shapes can still be double clicked and the text edited
- color of shape can still be changed
- maybe more

Reply:
Fixed.